Examples/Scripts/GenericfMRIVolumeProcessingPipelineBatch.sh

```diff
@@ -227,17 +227,20 @@ for Subject in $Subjlist ; do
 
         # Not using Siemens Gradient Echo Field Maps for susceptibility distortion correction
         # Set following to NONE if using TOPUP
-        MagnitudeInputName="NONE" #Expects 4D Magnitude volume with two 3D volumes (differing echo times)
-        PhaseInputName="NONE" #Expects a 3D Phase difference volume (Siemen's style)
-        DeltaTE="NONE" #2.46ms for 3T, 1.02ms for 7T
-
-        # Path to General Electric style B0 fieldmap with two volumes
-        #   1. field map in degrees
+        # or set the following inputs if using regular FIELDMAP (i.e. SiemensFieldMap GEHealthCareFieldMap PhilipsFieldMap)
+        MagnitudeInputName="NONE" #Expects 4D Magnitude volume with two 3D volumes (differing echo times) - or a single 3D Volume
```

Comment:
This may be confusing to users.  If supplying just a single 3D Volume, does it matter whether it is the magnitude image of the 1st or 2nd echo?

Reply:
GEHC outputs only the magnitude image of the echo 1. However, I kept the same philosophy as for siemens/philipps and if a 4D magnitude image if supplied, then it is averaged. 
I have added the "- or a single 3D Volume" to stress that the 4D magnitude was not mandatory. 

I understood that Siemens outputs the 2 magnitude images but not 100% sure.